### ISSUE TITLE
[visual-recognition] "Unauthorized: Access is denied due to invalid credentials" only on parallelised requests

### ISSUE THREAD

**Issue by holly-cummins** [watson-developer-cloud/node-sdk#698]
I'm seeing issues when sending parallelised requests (using Bluebird promise.map()) to Watson Visual Recognition. If I send a single request, all works fine. If I sent two requests at the same time, I get the response "Unauthorized: Access is denied due to invalid credentials". 

I'm not sure if this is more likely to be an SDK issue or a back end service issue.

**Comment by anweshan** [comment]
Looks like this issue is related to https://github.com/watson-developer-cloud/node-sdk/issues/688

Can you also try  removing `request.jar()` from the `base_service.js`? We will put an option in the instructor by end of the week to ignore cookies but you can try that fix for now

**Comment by holly-cummins** [comment]
Thanks very much for the speedy response. We tried both workarounds, without luck. We also tried reducing our parallelism to one, and that didn't help, so it's a bit peculiar. What we're seeing is the following: 
1. First request (jpg) - always succeeds. 
2. Second request (jpg) - 90% success rate
3. Third request (zip with 20 images in it) - 5% success rate

We see this even if we create a new instance of the VisualRecognitionV3 on the third request. 

We did notice our version of the client was quite old, so we've upgraded, and now have different intermittent errors (#699).
